lib/utils/schema/inspec.exec.full.json

```diff
@@ -0,0 +1 @@
+{"type":"object","additionalProperties":false,"properties":{"profiles":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"name":{"type":"string"},"version":{"type":"string","optional":true},"title":{"type":"string","optional":true},"maintainer":{"type":"string","optional":true},"copyright":{"type":"string","optional":true},"copyright_email":{"type":"string","optional":true},"license":{"type":"string","optional":true},"summary":{"type":"string","optional":true},"supports":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"os-family":{"type":"string","optional":true}}},"optional":true},"controls":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"id":{"type":"string"},"title":{"type":["string","null"]},"desc":{"type":["string","null"]},"impact":{"type":"number"},"refs":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"ref":{"type":"string"},"uri":{"type":"string","optional":true},"url":{"type":"string","optional":true}}}},"tags":{"type":"object"},"code":{"type":"string"},"source_location":{"type":"object","properties":{"ref":{"type":"string"},"line":{"type":"number"}}},"results":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"status":{"type":"string"},"code_desc":{"type":"string"},"run_time":{"type":"number"},"start_time":{"type":"string"},"skip_message":{"type":"string","optional":true},"resource":{"type":"string","optional":true}}}}}}},"groups":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"id":{"type":"string"},"title":{"type":"string","optional":true},"controls":{"type":"array","items":{"type":"string"}}}}},"attributes":{"type":"array"}}}},"statistics":{"type":"object","additionalProperties":false,"properties":{"duration":{"type":"number"}}},"version":{"type":"string"},"controls":"array","other_checks":"array"}}
```

Comment:
We certainly can :) I left it compressed initially to make sure this is just the minified schema. Reasoning should be done via the `schema.rb` file, changes too; These schemas get complex rather quicly where the `schema.rb` is easy to understand and follow. By making it more human-readable we give more access to it, but may also invite people to try to change the json file directly.

What do you think?